builds/jenkins-build-pack-website/Jenkinsfile

```diff
@@ -134,7 +134,13 @@ node()  {
 
 				echo "create_cloudfront_url====" + create_cloudfront_url
 				send_status_email('STARTED', '')
-				validateTemplate()
+				if(service_config['runtime'] == 'angular') {
```

Comment:
We should define a new key "framework" applicable to websites. Lets store this in catalog (dynamodb table:jazz_services) in metadata.